tools/accuracy_checker/accuracy_checker/annotation_converters/ms_coco.py

```diff
@@ -119,17 +133,20 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
             self.dataset_meta, full_annotation, self.use_full_label_map, self.has_background
         )
 
-        meta = {}
+        self.meta = {}
         if self.has_background:
             label_map[0] = 'background'
-            meta['background_label'] = 0
+            self.meta['background_label'] = 0
 
-        meta.update({'label_map': label_map})
+        self.meta.update({'label_map': label_map})
         detection_annotations, content_errors = self._create_representations(
             image_ids, annotations, label_id_to_label, check_content, progress_callback, progress_interval
         )
 
-        return ConverterReturn(detection_annotations, meta, content_errors)
+        return ConverterReturn(detection_annotations, self.meta, content_errors)
+
+    def meta_update(self, updates):
```

Comment:
why do you need this method?
you can directly update meta in your code without this method call

Reply:
Fixed